history/taskana-simplehistory-provider/src/main/java/pro/taskana/simplehistory/impl/ClassificationHistoryQueryImpl.java

```diff
@@ -415,32 +415,26 @@ public ClassificationHistoryQuery orderByCustomAttribute(int num, SortDirection
 
   @Override
   public List<ClassificationHistoryEvent> list() {
-    List<ClassificationHistoryEvent> result = new ArrayList<>();
+    List<ClassificationHistoryEvent> result;
     try {
-      taskanaHistoryEngine.openConnection();
-      result = taskanaHistoryEngine.getSqlSession().selectList(LINK_TO_MAPPER, this);
-      return result;
-    } catch (SQLException e) {
-      LOGGER.error(SQL_EXCEPTION_MESSAGE, e.getCause());
```

Comment:
That's a reasonable thought. However, the SQLException came from the openConnection() of the TaskanaHistoryEngine (which was removed in this PR), because the siganture had a throws SQLException in case that the setSchema() method propagated it. This Exception is now directly caught and logged in the openConnection of the TaskanaEngineImpl.